opensrp-tasking/src/test/java/org/smartregister/tasking/presenter/TaskingHomePresenterTest.java

```diff
@@ -0,0 +1,945 @@
+//    }
+//
+//    @Test
+//    public void testOnChangeInterventionStatusForPAOT() throws JSONException {
+//
+//        Feature mapboxFeature = initTestFeature("id1");
+//        Whitebox.setInternalState(presenter, "selectedFeature", mapboxFeature);
+//        presenter.onChangeInterventionStatus(PAOT);
+//        verify(view).showProgressDialog(R.string.fetching_paot_title, R.string.fetching_paot_message);
+//        verify(taskingHomeInteractor).fetchInterventionDetails(PAOT, "id1", true);
+//    }
+//
+//    @Test
+//    public void testOnFeatureSelectedByLongClickWhenStructureIsInactive() throws Exception {
+//
+//        Feature feature = initTestFeature("id1");
+//        feature.addStringProperty(LOCATION_STATUS, INACTIVE.name());
+//
+//        Whitebox.invokeMethod(presenter, "onFeatureSelectedByLongClick", feature);
+//        verify(view).displayToast(R.string.structure_is_inactive);
+//    }
+
+//    @Test
+//    public void testOnFeatureSelectedByLongClickWhenStructureHasNoTask() throws Exception {
+//
+//        Feature feature = initTestFeature("id1");
+//        feature.addStringProperty(LOCATION_STATUS, ACTIVE.name());
+//        feature.removeProperty(TASK_IDENTIFIER);
+//
+//        Whitebox.invokeMethod(presenter, "onFeatureSelectedByLongClick", feature);
+//        verify(view).displayMarkStructureInactiveDialog();
+//    }
+//
+//    @Test
+//    public void testOnFeatureSelectedByLongClickWhenTaskBusinessStatusIsNotVisited() throws Exception {
+//
+//        Feature feature = initTestFeature("id1");
+//        feature.addStringProperty(LOCATION_STATUS, ACTIVE.name());
+//        feature.addStringProperty(TASK_BUSINESS_STATUS, NOT_VISITED);
+//        feature.addStringProperty(TASK_IDENTIFIER, "task-1");
+//
+//        Whitebox.invokeMethod(presenter, "onFeatureSelectedByLongClick", feature);
+//        verify(view).displayMarkStructureInactiveDialog();
+//    }
+//
+//    @Test
+//    public void testOnFeatureSelectedByLongClickWhenTaskHasBeenCompleted() throws Exception {
+//
+//        Feature feature = initTestFeature("id1");
+//        feature.addStringProperty(LOCATION_STATUS, ACTIVE.name());
+//        feature.addStringProperty(TASK_BUSINESS_STATUS, SPRAYED);
+//        feature.addStringProperty(TASK_IDENTIFIER, "task-1");
+//
+//        Whitebox.invokeMethod(presenter, "onFeatureSelectedByLongClick", feature);
+//        verify(view).displayToast(R.string.cannot_make_structure_inactive);
+//    }
+
+
+    @Test
+    public void testStartFormShouldPopulateFormData() throws Exception {
+        when(jsonFormUtils.getFormName(null, IRS)).thenReturn(SPRAY_FORM_ZAMBIA);
+        when(jsonFormUtils.getFormJSON(view.getContext(), SPRAY_FORM_ZAMBIA, feature, null, null)).thenReturn(new JSONObject(TestingUtils.DUMMY_JSON_FORM_STRING));
+        presenter.startForm(feature, null, IRS);
+//        verify(jsonFormUtils).populateForm(any(), any());
```

Comment:
I am not sure of the intention when commenting these out. Could you do something about it? Create an issue, todo, remove, fix